<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>531</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>461</width>
    <height>372</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QGridLayout" name="central_layout">
    <property name="verticalSpacing">
     <number>25</number>
    </property>
    <item row="1" column="7">
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="Result" name="result_widget" native="true"/>
      </item>
     </layout>
    </item>
    <item row="1" column="5">
     <layout class="QGridLayout" name="option_layout">
      <property name="verticalSpacing">
       <number>-1</number>
      </property>
      <item row="0" column="0">
       <widget class="Option" name="option_widget" native="true"/>
      </item>
     </layout>
    </item>
    <item row="1" column="6">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="Option" name="progress_widget" native="true"/>
      </item>
     </layout>
    </item>
    <item row="0" column="5" colspan="3">
     <layout class="QGridLayout" name="gridLayout_5">
      <property name="verticalSpacing">
       <number>25</number>
      </property>
      <item row="1" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Enter Root Directory...&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Setting" name="setting_widget" native="true"/>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="title_label">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:48pt;&quot;&gt;DataMan&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Merry Christmas Daddy Doo! &lt;br&gt; Hope this helps at work! &lt;br&gt;Designed by David Chalco (12-29-2015)&lt;br&gt;Chal Co. 2015&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>740</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Option</class>
   <extends>QWidget</extends>
   <header>option.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Setting</class>
   <extends>QWidget</extends>
   <header>setting.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Result</class>
   <extends>QWidget</extends>
   <header>result.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
